fingertip_keyboard_input/mouse_and_keyboard_control.py

```diff
@@ -334,59 +345,109 @@ def get_filtered_center(center, t):
                     if (time_stamp - time_start > 1):
                         time_start = time_stamp
                         if right_gesture == "one": #switch to previous app
-                            pyautogui.hotkey('command', 'tab')
+                            # pyautogui.hotkey('command', 'tab')
+                            # added Windows shortcut commands
+                            if system_type == "Mac":
+                                pyautogui.hotkey('command', 'tab')
+                            elif system_type == "Windows":
+                                pyautogui.hotkey('alt', 'tab')
                         elif right_gesture == "two": #browse windows
-                            pyautogui.hotkey('ctrl', 'up')
+                            # pyautogui.hotkey('ctrl', 'up')
+                            if system_type == "Mac":
+                                pyautogui.hotkey('ctrl', 'up')
+                            elif system_type == "Windows":
+                                pyautogui.hotkey('ctrl', 'alt', 'tab')
                         elif right_gesture == "three": #minimize active window
-                            pyautogui.hotkey('command', 'm')
-                elif mode == 'safari':
+                            # pyautogui.hotkey('command', 'm')
+                            if system_type == "Mac":
+                                pyautogui.hotkey('command', 'm')
+                            elif system_type == "Windows":
+                                pyautogui.hotkey('super', 'm')
+                #changed safari mode to browser mode
+                elif mode == 'browser':
                     if time_start is None:
                         time_start = time.time()
                     time_stamp = time.time()
                     if (time_stamp - time_start > 1):
```

Comment:
I'm open to what you think, but I'm not sure we should use the "timer" that the previous team implemented. It seems to lag a lot. Instead, we might prefer using "prev_gesture" for the "rising edge" of a gesture. For example: `if right_gesture == 'one' and prev_right_gesture != 'one':`

Reply:
Yeah that makes sense, will try to change to prev_gesture.